exercises/concept/language-list/.meta/examples/languageList.example.u

```diff
@@ -0,0 +1,26 @@
+languageList.new : [Text]
+languageList.new = []
+
+languageList.add : Text -> [Text] -> [Text]
+languageList.add input list = input +: list
+
+languageList.drop1 : [Text] -> [Text]
+languageList.drop1 = cases
+  head +: tail -> tail
+  [] -> []
+
+languageList.size : [Text] -> Nat
+languageList.size list =
+  go acc = cases
+    [] -> acc
+    h :+ t -> go (Nat.increment acc) t
+  go 0 list
```

Comment:
This uses recursion, which makes a lot of sense. That said, recursion is likely a concept you'd want to have as a prerequisite for this exercise, so it should probably be added as such to the `config.json` file.

Reply:
In the same vein, this example also uses pattern matching, which you also want as a prerequisite here. In F#, the way we solved this is by having one introductory `pattern-matching` exercise: https://github.com/exercism/fsharp/tree/main/exercises/concept/guessing-game
This exercise introduces basic pattern matching, and then the lists exercise builds on that by showing how to do pattern matching with lists specifically.